templating/templating_app/templates/templating_app/filters.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'templating_app/index.html' %}
+
+{% block title %}Filters{% endblock %}
+
+{% block content %}
+<h1>Django filter examples</h1>
+
+<h2>Add</h2>
+{{ 6|add:"6" }}
+
+<h2>Date</h2>
+<p>It is {% now "l\, F jS" %}.</p>
```

Comment:
I'd _love_ if you'd give the examples of a few different date formats.  Date formatting is something I wish I were better at, but at least I know where to look for information.  I'd like you to be able to do the same.